src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { snazzyButtonStyle } from './helpers';
+
+
+interface ButtonProps {
+  children: React.ReactNode;
+}
+
+const styles = {
+  snazzyButton: {
+
+  }
+};
+
+const useStyles = createUseStyles(styles);
+
+interface SnazzyButtonProps {
+  isSnazzy: boolean;
+}
+
+const SnazzyButton: React.FC<SnazzyButtonProps> = ({ isSnazzy, children, ...rest }) => {
+  const classes = useStyles();
+
+  const classNames = {
+    [classes.snazzyButton]: isSnazzy,
+
+  };
+
+  const className = Object.keys(classNames)
+    .filter((key) => classNames[key])
+    .join(' ');
```

Comment:
this really shouldn't be necessary, you had code that worked really well for enabling/disabling the "isSnazzy" prop earlier. 
Remove this, I'll show you what I mean in another comment.